<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainwindow1</class>
 <widget class="QWidget" name="mainwindow1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Вход</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="1" column="0">
        <widget class="QLineEdit" name="login_line">
         <property name="styleSheet">
          <string>
         border-radius: 10px;
         padding: 5px;
        </string>
         </property>
         <property name="placeholderText">
          <string>Login</string>
         </property>
         <property name="cursorMoveStyle">
          <enum>Qt::VisualMoveStyle</enum>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="info_check_label">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="7" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="button_enter">
         <property name="styleSheet">
          <string notr="true">QPushButton{
	color: rgb(61, 140, 62);
       background-color: #4CAF50;
         color: white;
         border: none;
         border-radius: 10px;
         padding: 10px;
}
          QPushButton:hover {
              background-color: #3d8c3e;
          }
        </string>
         </property>
         <property name="text">
          <string>Вход</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>login_icon.png</normaloff>login_icon.png</iconset>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLineEdit" name="pass_line">
         <property name="styleSheet">
          <string>
         border-radius: 10px;
         padding: 5px;
        </string>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
         <property name="placeholderText">
          <string>Password</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="info_label_enter">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Регистрация</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0">
        <widget class="QLineEdit" name="reg_pass_line">
         <property name="styleSheet">
          <string>
         border-radius: 10px;
         padding: 5px;
        </string>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
         <property name="placeholderText">
          <string>Password</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="info_label_reg">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="button_reg">
         <property name="styleSheet">
          <string notr="true">QPushButton{
         background-color: #2196F3;
         color: white;
         border: none;
         border-radius: 10px;
         padding: 10px;
  }
          QPushButton:hover {
              background-color: #1a74ba;
          }</string>
         </property>
         <property name="text">
          <string>Регистрация</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>register_icon.png</normaloff>register_icon.png</iconset>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLineEdit" name="reg_login_line">
         <property name="styleSheet">
          <string>
         border-radius: 10px;
         padding: 5px;
        </string>
         </property>
         <property name="placeholderText">
          <string>Login</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
